includes/status.include

```diff
@@ -11,5 +11,5 @@ per [W3C Process - 6.4 Candidate Recommendation](https://www.w3.org/2017/Process
 
 <ul>
  <li>The <{autocapitalize}> attribute</li>
+ <li>The <{registerContentHandler()}> method, and related methods like like <{unregisterContentHandler()}> and <{isContentHandlerRegistered()}></li>
```

Comment:
remove "like like" ...
The at risk stuff should be clear about what is at risk, besides the duplication :)

Otherwise LGTM, go ahead and merge...